core/unit_test/TestViewAPI.hpp

```diff
@@ -1542,6 +1542,7 @@ class TestViewAPI {
   }
 
   static void run_test_error() {
+#if !((HIP_VERSION_MAJOR == 5) && (HIP_VERSION_MINOR == 3))
```

Comment:
No, instead guard https://github.com/kokkos/kokkos/blob/4271bd9f034f028c9d93691b9506f62063f67f01/core/unit_test/TestViewAPI_d.hpp#L29-L31
and make sure you add a comment

Reply:
That was for debugging. It shouldn't be there anymore